### ISSUE TITLE
Toggle switch UI not showing properly.

### ISSUE THREAD

**Issue by karanMalhii** [themesberg/flowbite-react#1360]
on latest version 8.0 the toggle switch button is not working; properly.

<img width="914" alt="image" src="https://github.com/themesberg/flowbite-react/assets/122113953/0b9ddae7-664f-4e76-8c33-b6dcfaaed9c1">

https://flowbite-react.com/docs/components/forms#toggle-switch

**Comment by SutuSebastian** [comment]
Do u mean "its NOT working properly" maybe? 😄

**Comment by karanMalhii** [comment]
> Do u mean "its NOT working properly" maybe? 😄

nice catch😉

**Comment by karanMalhii** [comment]
now it seems working fine.

**Comment by SutuSebastian** [comment]
Still broken in the docs tho.

**Comment by taskrider2022** [comment]
Also broken if you use them in your project

**Comment by alfreddagenais** [comment]
I just mentioned into this issue https://github.com/themesberg/flowbite-react/issues/1321#issuecomment-2032778546 A temporary fix for me : 

```javascript
export const customThemeToggleSwitch: CustomFlowbiteTheme['toggleSwitch'] = {
    toggle: {
      base: "after:rounded-full rounded-full border group-focus:ring-4 group-focus:ring-cyan-500/25",
      checked: {
          on: "after:bg-white after:translate-x-full",
          off: "after:bg-gray-400 dark:after:bg-gray-500 border-gray-200 bg-gray-200 dark:border-gray-600 dark:bg-gray-700",
      },
    },
};
``` 

This solution might help others facing similar problems.

**Comment by MateoWartelle** [comment]
#1397 - you can close

**Comment by SutuSebastian** [comment]
Fixed in https://github.com/themesberg/flowbite-react/pull/1428

**Comment by SutuSebastian** [comment]
Fixed in [flowbite-react@0.10.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.10.0) release.
